src/graphql/config.ts

```diff
@@ -1,17 +1,13 @@
+import DataLoader from 'dataloader'
+import { JsonAPI } from './search/cms/normalize'
+import { RootObject as OpenAPI } from '../fixtures/openapi/types'
+
 export type Context = {
   loaders: {
-    cms: {
-      load: Function
-      clear: Function
-    }
-    data: {
-      load: Function
-      clear: Function
-    }
-    datasets: {
-      load: Function
-      clear: Function
-    }
+    cms: DataLoader<string, PromiseSettledResult<JsonAPI>, string>
+    data: DataLoader<string, PromiseSettledResult<{ count: number; results: object[] }>, string>
```

Comment:
Do we not have a better type than `object[]`?

Reply:
As soon as I will add the fixtures for data search API's in atlas, I will create a separate PR that will type this (or let's wait when the generic API Client is made 😄 )